gallery/src/components/NxToggle/NxTogglePage.tsx

```diff
@@ -10,10 +10,8 @@ import { NxTextLink } from '@sonatype/react-shared-components';
 import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
 
 import NxToggleExample from './NxToggleExample';
-import NxToggleGaplessExample from './NxToggleGaplessExample';
 
-const exampleCode = require('./NxToggleExample?raw'),
-    exampleGaplessCode = require('./NxToggleGaplessExample?raw');
+const exampleCode = require('./NxToggleExample?raw');
 
 const NxTogglePage = () =>
   <>
```

Comment:
Text a few lines down needs to be updated: "Child VDOM will be used as a label preceeding the toggle control."

Reply:
> Text a few lines down needs to be updated: "Child VDOM will be used as a label preceeding the toggle control."

https://github.com/sonatype/sonatype-react-shared-components/pull/410/commits/584e1116291272c6f86f598fb18d95a4f4b2797c